scopus-import/src/main/java/no/sikt/nva/scopus/conversion/NvaCustomerConnection.java

```diff
@@ -36,7 +36,7 @@ public boolean isNvaCustomer(CristinOrganization cristinOrganization) {
                    .map(this::fetchResponse)
                    .map(HttpResponse::statusCode)
                    .map(NvaCustomerConnection::isHttpOk)
-                   .orElseThrow();
```

Comment:
Not sure if I agree on ignoring the exception and return false. This API is our own api, it should be up and running at all times.

Reply:
Endrer tilbake til exception